spinalcordtoolbox/deepseg/models.py

```diff
@@ -76,6 +76,14 @@
         "description": "Grey/white matter seg on exvivo human T2w.",
         "contrasts": ["t2"],
         "default": False,
+    },
+    "model_7T_multiclass_gm_sc_unet2d": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_gm-wm_t2star_7T_unet3d-multiclass/blob/main/Model_SC_GM_7T.zip"
```

Comment:
```suggestion
            "https://github.com/ivadomed/model_seg_gm-wm_t2star_7t_unet3d-multiclass/blob/main/Model_SC_GM_7T.zip"
```
i suggest to keep everything in small letter for consistency. This implies: renaming the repos